IE compatibility

backflip opened the pull request:
With some small changes we can make this work in IE 10+. 🎉

Based on @mikejolley's excellent article: https://mikejolley.com/2019/08/02/building-a-cross-browser-compatible-multi-handle-range-slider/

Limitation: In some situations, the inputs are switched (see https://github.com/LeaVerou/multirange/pull/43). However, slider changes are animated in IE, so the switch is visible and feels slightly weird. Doesn't affect the functionality, though.

LeaVerou commented:
This is great work, thank you! But do we really need the extra span? IT looks backwards incompatible with existing usage 😢

backflip commented:
@LeaVerou, good point. I reverted the markup changes and added custom IE-styles for the demo instead (fallback for not supporting `width: max-content;` and not handling `text-align: center;`).

LeaVerou commented:
LGTM, merging, thanks!
